### PR TITLE
Added `email-header: decode" operator.

### PR DESCRIPTION
**Pull request by spearmin10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Added `email-header: decode" operator.

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/ExtFilter/ReleaseNotes/1_1_0.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
 #### Playbooks
```
Comment (Itay4) [inline Packs/ExtFilter/ReleaseNotes/1_1_0.md:1]: Please revert the changes in this file.

File: Packs/ExtFilter/ReleaseNotes/1_1_1.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+##### Test - ExtFilter Main
+- Modified task titles
+
+#### Scripts
+##### ExtFilter
+- Added `email-header: decode`
```
Comment (Itay4) [inline Packs/ExtFilter/ReleaseNotes/1_1_1.md:8]: Please add here to docker image bump.

File: Packs/ExtFilter/Scripts/ExtFilter/ExtFilter.py [hunk 3]
```diff
@@ -1534,6 +1535,28 @@ def filter_value(
...
+                    elif isinstance(decoded_s, bytes):
+                        out += decoded_s.decode('utf-8')
+                    else:
+                        out += decoded_s
+            except Exception:
+                for encoding in encoding_types:
+                    try:
+                        out = lstr.decode(encoding)
+                        break
+                    except: # noqa: E722
```
Comment (Itay4) [inline Packs/ExtFilter/Scripts/ExtFilter/ExtFilter.py]: Please consider logging the exception messages using `demisto.debug`

File: Packs/ExtFilter/ReleaseNotes/1_1_1.md (hunk 2 above)
Comment (Itay4) [inline Packs/ExtFilter/ReleaseNotes/1_1_1.md:8]: I think we can improve the release note entry here to mention that this is an operation we added here

File: Packs/ExtFilter/ReleaseNotes/1_1_1.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+
+#### Playbooks
+##### Test - ExtFilter Main
+- Modified task titles
```
Comment (Itay4) [inline Packs/ExtFilter/ReleaseNotes/1_1_1.md:4]: After moving the test playbook to the right directory, you can remove it from the changelog

**Review by Itay4 — COMMENTED** [review]
Looks good overall
Added some minor comments
Also, seems like the test playbook is released as a playbook, even though it shouldn't - please move it to the `TestPlaybooks` directory

File: Packs/ExtFilter/ReleaseNotes/1_1_0.md (hunk 1 above)
Comment (spearmin10) [reply Packs/ExtFilter/ReleaseNotes/1_1_0.md]: Reverted.

File: Packs/ExtFilter/ReleaseNotes/1_1_1.md (hunk 2 above)
Comment (spearmin10) [reply Packs/ExtFilter/ReleaseNotes/1_1_1.md:9]: Added it.

Comment (spearmin10) [reply Packs/ExtFilter/ReleaseNotes/1_1_1.md:9]: Added it.

**Comment by spearmin10** [comment]
@Itay4 Everything is fixed. Please review again.

**Comment by Itay4** [comment]
> @Itay4 Everything is fixed. Please review again.

I'd still like us to move the test playbook to the right directory, is there a reason you haven't done so? I can help with it if needed

**Comment by spearmin10** [comment]
@Itay4 I've moved them to TestPlaybooks.

File: Packs/ExtFilter/ReleaseNotes/1_1_1.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+- Added test tasks for `email-header: decode`
+- Added test tasks for `regex: replace`
```
Comment (Itay4) [inline Packs/ExtFilter/ReleaseNotes/1_1_1.md:6]:
```suggestion
```

**Review by Itay4 — APPROVED** [review]
